spec/features/trainees/creating_a_new_trainee_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "rails_helper"
+
+RSpec.feature "Create trainee journey" do
+  let(:index_page) { PageObjects::Trainees::IndexPage.new }
```

Comment:
I think we've opted to using instance variables than the lets. ie https://github.com/DFE-Digital/register-trainee-teacher-data/blob/e1e292500a2681665d9310e6a1133d8f73165366/spec/features/trainees/trainee_summary_spec.rb#L20